aws/services/acm/id.ftl

```diff
@@ -32,6 +32,10 @@
 [#function formatDomainCertificateId certificateObject, hostName=""]
     [#local primaryDomain = getCertificatePrimaryDomain(certificateObject) ]
     [#if primaryDomain.Name?has_content ]
+        [#if !primaryDomain.Name?contains(".") ]
+            [@fatal message="Missing domain name for ACM primary domain" context=solution /]
```

Comment:
Doesn't look like solution is available at this point? 
Is it worth performing this validation in getCertificatePrimaryDomain instead as that is more where the issue is occurring?

Reply:
Moved to getCertificatePrimaryDomain